package.json

```diff
@@ -133,8 +133,8 @@
 				"category": "Sourcery"
 			},
 			{
-				"command": "sourcery.hub.open",
-				"title": "Open Sourcery Hub",
+				"command": "sourcery.hub.start",
+				"title": "Start Sourcery Hub",
```

Comment:
<!-- Polish comments are like a suggestion, where there is nothing necessarily wrong with the relevant content, there's just some ways to immediately improve the quality. -->
**💄 polish** I'd sugg
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏I'd suggest keeping this as `Open` as from the point of view of the user it opens the hub in VS Code